### PR TITLE
Add model signature to served model

### LINKED ISSUE #35: Add model signature stored in MLmodel file
Add definition of input and output data of served model by manual definition in model creation process

### PR DESCRIPTION
**Pull request by avan1235**
Closes #35. For now we have to work with this example model as iris classification is not working for me (which is connected with [this](https://github.com/mlflow/mlflow/issues/2544) bug)

### CODE REVIEW

File: dev-environment/mlflow/Dockerfile [hunk 1]
```diff
@@ -2,21 +2,20 @@ FROM conda/miniconda3:latest
...
 RUN conda install -c anaconda jupyter=1.0.0
-RUN conda install -c conda-forge mlflow=1.8.0
 RUN conda install -c anaconda scikit-learn=0.22.1
 RUN conda install -c anaconda psycopg2=2.8.5
 RUN conda install -c anaconda boto3=1.14.12
-RUN conda install -c conda-forge azure-core=1.8.2
+RUN conda install -c conda-forge azure-core=1.9.0
 RUN conda install -c conda-forge azure-storage=0.36.0
-RUN conda install -c conda-forge azure-storage-blob=12.5.0
+RUN conda install -c conda-forge azure-storage-blob=12.6.0
```
Comment (kantoniak) [inline dev-environment/mlflow/Dockerfile:17]: Why do we need Azure services? We don't keep anything in Azure Blob Storage. Is `wine` dataset in some public Azure storage?

Comment (avan1235) [reply dev-environment/mlflow/Dockerfile:17]: not sure, it was needed when doing the setup of environment, now only the version updated. Dataset is at http://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-red.csv

Comment (kantoniak) [reply dev-environment/mlflow/Dockerfile:17]: Let's check if mflow works without these dependencies.

**Review by kantoniak — COMMENTED** [review]
Let's check if mflow works without Azure dependencies.

File: dev-environment/mlflow/Dockerfile (hunk 1 above)
Comment (avan1235) [reply dev-environment/mlflow/Dockerfile:17]: no it doesn't. I had to add it manually (it's not an example copied from some webpage). Its needed to serve model using the MLmodel definition stored on S3

**Comment by avan1235** [comment]
For me it didn't work, if you want to experiment you can PR such changes but I'm not sure if it is possible not to use these dependencies

**Review by jkukowski — APPROVED** [review]

**Pull request merged**